### ISSUE TITLE
[bug] skipping forward on paused videos doesn't work

### ISSUE THREAD

**Issue by toxicologist** [TeamNewPipe/NewPipe#5700]
### Checklist

- [x] I am using the latest version - x.xx.x 
- [ x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Actual behaviour

It only skips forward in the video when you unpause it. While the video is paused you cant skip forward or backwards

### Expected behavior

Should skip times regardless of the video being paused or not

### Screenshots/Screen recordings

I attached a video showing the bug

### Device info

 - Android version/Custom ROM version: 9
 - Device model: galaxy s8

**Comment by iamthesenate1** [comment]
For me it works perfectly

**Comment by iamthesenate1** [comment]
It works, but when it is paused and you skip forward it doesn't show you the time it skipped at, until you start the video.
You should close this.

**Comment by toxicologist** [comment]
> It works, but when it is paused and you skip forward it doesn't show you the time it skipped at, until you start the video.
> You should close this issue.

Yes, that's exactly the problem, you can't see what time the video is at, so it's hard to skip to the correct time. That is a bug and interferes with usability

**Comment by iamthesenate1** [comment]
Agree, but you should open another issue describing your problem better. The title leads the reader to think this is about a much bigger problem. What you are asking is, in my opinion more like an enhancement, rather than a bug.

**Comment by toxicologist** [comment]
It is a bug, i wont close the issue. But i will change the title to make it more descriptive. Thanks for the suggestion :)
